packages/mds-audit/api.ts

```diff
@@ -521,10 +504,21 @@ function api(app: express.Express): express.Express {
             if (start_time && end_time) {
               const deviceEvents = await readEvents(device.device_id, start_time, end_time)
               const deviceTelemetry = await readTelemetry(device.device_id, start_time, end_time)
-
+              const providerEvent = await db.readEventsWithTelemetry({
```

Comment:
Just curious why the provider event is being recalculated on each `GET` as opposed to being stored with the audit with the original `POST`.

Reply:
Doing it this way just felt a little safer and cleaner to me.

I wanted to avoid associating the event with the audit in the DB in case the audit or event changes after it was created. For example, maybe the event that should have been associated with the audit comes in late for some reason, or maybe we have a production issue and have to backfill events or audits, or maybe we need to change the start time of the audit for some reason. This way we always associate the right event with the audit based on the current data. It also keeps the audit table a little cleaner.

Reply:
SGTM